bids/grabbids/sample_validation_config.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
This file should probably go in a new `data` folder inside the layout module's tests folder. I.e., `bids/layout/tests/data` (note that `grabbids` has now been replaced by `layout` thanks to @ltirrell's PR).